syntax = "proto3";
package spacepi.messaging;

message EncapsulatedMessage {
    uint64 messageID = 1;
    uint64 instanceID = 2;
    bytes payload = 3;
}
